admin/app/dfp/DataMapper.scala

```diff
@@ -2,137 +2,19 @@ package dfp
 
 import com.google.api.ads.admanager.axis.v202502._
 import common.dfp._
-import dfp.ApiHelper.{isPageSkin, optJavaInt, toJodaTime, toSeq}
+import dfp.ApiHelper.{toJodaTime, toSeq}
 
 // These mapping functions use libraries that are only available in admin to create common DFP data models.
 class DataMapper(
-    adUnitService: AdUnitService,
-    placementService: dfp.PlacementService,
     customTargetingService: dfp.CustomTargetingService,
     customFieldService: dfp.CustomFieldService,
 ) {
-
   def toGuAdUnit(dfpAdUnit: AdUnit): GuAdUnit = {
     val ancestors = toSeq(dfpAdUnit.getParentPath)
     val ancestorNames = if (ancestors.isEmpty) Nil else ancestors.tail.map(_.getName)
     GuAdUnit(dfpAdUnit.getId, ancestorNames :+ dfpAdUnit.getName, dfpAdUnit.getStatus.getValue)
   }
 
-  private def toGuTargeting(session: SessionWrapper)(dfpTargeting: Targeting): GuTargeting = {
-
-    def toIncludedGuAdUnits(inventoryTargeting: InventoryTargeting): Seq[GuAdUnit] = {
-
-      // noinspection MapFlatten
-      val directAdUnits =
-        toSeq(inventoryTargeting.getTargetedAdUnits).map(_.getAdUnitId).map(adUnitService.activeAdUnit).flatten
-
-      // noinspection MapFlatten
-      val adUnitsDerivedFromPlacements = {
-        toSeq(inventoryTargeting.getTargetedPlacementIds).map(placementService.placementAdUnitIds(session)).flatten
-      }
-
-      (directAdUnits ++ adUnitsDerivedFromPlacements).sortBy(_.path.mkString).distinct
-    }
-
-    def toExcludedGuAdUnits(inventoryTargeting: InventoryTargeting): Seq[GuAdUnit] = {
-      toSeq(inventoryTargeting.getExcludedAdUnits).map(_.getAdUnitId).flatMap(adUnitService.activeAdUnit)
-    }
-
-    def toCustomTargetSets(criteriaSets: CustomCriteriaSet): Seq[CustomTargetSet] = {
-
-      def toCustomTargetSet(criteria: CustomCriteriaSet): CustomTargetSet = {
-
-        def toCustomTarget(criterion: CustomCriteria) =
-          CustomTarget(
-            customTargetingService.targetingKey(session)(criterion.getKeyId),
-            criterion.getOperator.getValue,
-            criterion.getValueIds.toSeq map (valueId =>
-              customTargetingService.targetingValue(session)(criterion.getKeyId, valueId)
```

Comment:
I think having removed these `customTargetingService` they are now redundant in other part of the code:
https://github.com/guardian/frontend/pull/27968/files#diff-cf9809602149f86e412482201118233cd6061e1f03672b10acafd8a962948448L59
and
https://github.com/guardian/frontend/pull/27968/files#diff-39686f1f0994434fd14262d6232c45718ee6a37e7325760fcf2fb64113b5a637L11

Having looked through the I can see it being used only as a statement builder?
https://github.com/guardian/frontend/blob/81a5a4392487e23ce0613f84b43029bd5fdf2a87/admin/app/dfp/SessionWrapper.scala#L58-L65

https://github.com/guardian/frontend/blob/81a5a4392487e23ce0613f84b43029bd5fdf2a87/admin/app/dfp/ServicesWrapper.scala#L17